src/vs/workbench/services/keybinding/browser/keybindingService.ts

```diff
@@ -906,13 +907,18 @@ class KeybindingsJsonSchema {
 		this.commandsEnumDescriptions.length = 0;
 
 		const knownCommands = new Set<string>();
-		const addKnownCommand = (commandId: string, description?: string | undefined) => {
+		const addKnownCommand = (commandId: string, description?: ILocalizedString | ICommandHandlerDescription | undefined) => {
```

Comment:
@ulugbekna I saw your name in the blame of these changes. Can you review from your perspective here?

Reply:
Looks good to me.